### ISSUE TITLE
Openchange new version?

### ISSUE THREAD

**Issue by BWF-IT** [openchange/openchange#418]
Hi

When can we expect the release of a new version of openchange? it has been almost a year since the last release (v 2.3). To me it seems like it is barely being worked on.

**Comment by simonsays42** [comment]
Indeed, the project seems to be dead. We definitely need a FORK right now, otherwise openchange will be no more than a theoretical case study. This neat piece of software has to be taken to PRODUCTION level (and we all know it still has a long way to go). Hopefully some techies here pick it up and improve on it (like adding essential features which make it a REAL exchange replacement in the NEAR future). Anyways I am having a hard time believing that this will happen any time soon.....

**Comment by SlavekB** [comment]
You can look at Zentyal fork, which is active in development:
https://github.com/zentyal/openchange/

**Comment by BWF-IT** [comment]
Unfortunately the Zentyal development is really slow, they havn't released a new version since November last year. As far as I can see, Opencahnge development has stalled because of Exchange moving from MAPI-RPC-HTTPS (Outlook Anywhere) to MAPI-HTTPS (Exchange Activesync) which is the default from Exchange/Outlook 2013 SP1 onwards. The problem is that there are still many businesses out there that use older versions of Outlook that does not support EAS.

The other motivation for a fork would be the fact that not everyone would want to use zentyal, the might want to use aonther distro as the base for their server.

**Comment by SlavekB** [comment]
Currently latest release 2.4-zentyal23 is from 5 April 2016, previous 2.4-zentyal22 is from 22 Februar 2016,... it does not look like slow development. See:
  https://github.com/zentyal/openchange/releases

It is not necessary to use OpenChange from Zentyal fork only with zentyal server. Is it possible to build and use it separately ... for example with SOGo packages. See:
  https://lists.inverse.ca/sogo/arc/users/2016-07/msg00253.html

IMO, the best solution is to avoid OpenChange, avoid ActiveSync and use a plugin that adds into MS Outlook  support for CalDAV and CardDAV. See:
  https://sourceforge.net/projects/outlookcaldavsynchronizer/

**Comment by abartlet** [comment]
Because of the linkage and history between [Samba](https://samba.org) and OpenChange, this question comes up in Samba occasionally (I'm on the [Samba Team](https://samba.org/samba/team)).

We recently added [explanation page on OpenChange status](https://wiki.samba.org/index.php/OpenChange) to the [Samba wiki](https://wiki.samba.org).

**Comment by SlavekB** [comment]
There are patches in my fork that allow to build OpenChange with Samba 4.5. For now, I did not test whether building with a newer Samba version is also possible, or whether it requires additional patches. However, my patches are not a new start of development - just a little maintenance.

See https://github.com/SlavekB/openchange

**Comment by Neustradamus** [comment]
@SlavekB Have you look for Samba 4.7+?

**Comment by kloczek** [comment]
It would be nice good to have maintainer comment about current situation and possible new release and/or project current situation .. please.

**Comment by bradh** [comment]
As an ex-developer, I believe there is no status.

**Comment by jelmer** [comment]
@kloczek the lead developer has moved on, and nobody from the original team has worked on OpenChange in at least two years. No releases or other updates are planned.

Friendly forks like @SlavekB 's are probably your best bet if you'd like to run OpenChange.

**Comment by kloczek** [comment]
If you are no longer interested and/or have no time to maintain openchange maybe it would be transfer repo to  @SlavekB if he would like to continue work on that?

**Comment by jelmer** [comment]
It would be great if @SlavekB kept working on their OpenChange fork, but write access to this repository is not a prerequisite for that - git means that this repository is no more special than any other. If their fork picked up, people would find it. 

@SlavekB 's last change appears to be from 2017 so I don't think they're particularly active though.
